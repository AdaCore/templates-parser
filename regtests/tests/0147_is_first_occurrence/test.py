from test_support import *

run('testme', ["ifo.tmplt"])
