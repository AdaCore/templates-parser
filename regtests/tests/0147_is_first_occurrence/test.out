
1: display
a value
2: one / display
a value
3: one / empty
4: two / display
a value
5: two / empty
6: display
a value
7: one / empty
8: two / empty

Loop:
a table
with
many
values
to
be
displayed
one
by

